src/nfa/limex_exceptional.h

```diff
@@ -129,7 +129,7 @@ int RUN_EXCEPTION_FN(const EXCEPTION_T *e, STATE_ARG,
         union RepeatControl *repeat_ctrl = ctx->repeat_ctrl + info->ctrlIndex;
         char *repeat_state = ctx->repeat_state + info->stateOffset;
 
-        if (unlikely(e->trigger == LIMEX_TRIGGER_POS)) {
```

Comment:
These branch predictions can improve the performance and are not recommended to be deleted.